sql/2019/01_JavaScript/01_02a.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# 01_02a: Distribution of 1P/3P JS bytes
+SELECT
+  percentile,
+  ROUND(APPROX_QUANTILES(first_party, 1000)[OFFSET(percentile * 10)], 2) AS first_party_js_kbytes,
+  ROUND(APPROX_QUANTILES(third_party, 1000)[OFFSET(percentile * 10)], 2) AS third_party_js_kbytes
+FROM (
+  SELECT
+    SUM(IF(NET.HOST(page) = NET.HOST(url), respSize, 0) / 1024) AS first_party,
+    SUM(IF(NET.HOST(page) != NET.HOST(url), respSize, 0) / 1024) AS third_party
```

Comment:
would you be interested in using `lighthouse-infrastructure.third_party_web.2019_07_01_all_observed_domains` at all?

Reply:
Hmm that might not be a bad idea. I'll play with that dataset and see how much the results differ.

Reply:
Updated